### ISSUE TITLE
Programmatically set the selected NavItem

### ISSUE THREAD

**Issue by GiambaFr** [trendmicro-frontend/react-sidenav#68]
Can you provide a function to set the selected NavItem from Nav or SideNav's element to set "active" the right one, even if the user either enters the pathname himself or does history.back()?

Two NavItem can be active when doing:
<Nav defaultSelected={location.pathname.substring(1)} >
   <NavItem eventKey="home" **active={"home" == location.pathname.substring(1) ? true: false}>**
                              <NavIcon>
                                  <i className="fa fa-home fa-2x" title={t('home')} />
                              </NavIcon>
                              <NavText>
                                  {t('Home')}
                              </NavText>
                          </NavItem>
                          <NavItem eventKey="user" **active={"user" == location.pathname.substring(1) ? true: false}**>
                              <NavIcon>
                                  <i className="fa fa-fw fa-user fa-2x" title={t('Personal informations')} />
                              </NavIcon>
                              <NavText>
                                {t('Personal informations')}
                              </NavText>
                          </NavItem>

Thanks

**Comment by Liam-OShea** [comment]
Were you able to find a solution to this @GiambaFr ?

**Comment by GiambaFr** [comment]
No, I've played with it a few hours to finaly figure out it was not the lib I was looking for. It was not open enough for the uses I needed to complete my project. So I moved to React.Router and I build it myself.
